pandas-stubs/core/frame.pyi

```diff
@@ -247,18 +223,35 @@ class DataFrame:
     ) -> None: ...
     @overload
     def sort_values(
-        self,
-        by: List[_str],
-        inplace: Optional[Literal[False]] = ...,
-        axis: _AxisType = ...,
-        ascending: bool = ...,
+        self, by: List[_str], inplace: Optional[Literal[False]] = ..., axis: _AxisType = ..., ascending: bool = ...,
     ) -> DataFrame: ...
     def std(self) -> DataFrame: ...
     @overload
     def sum(self) -> Series: ...
     @overload
     def sum(self, axis: _AxisType) -> Series: ...
-    def to_csv(self, filename: Union[Path, _str], index: bool = ...) -> None: ...
+    def to_csv(
+        self,
+        path_or_buf: Optional[Union[Path, _str]] = ...,
+        sep: _str = ...,
+        na_rep: _str = ...,
+        float_format: Union[_str, None] = ...,
+        columns: Union[Sequence[Union[Hashable, None]], None] = ...,
+        header: Union[bool, List[_str]] = ...,
+        index: bool = ...,
+        index_label: Union[bool, _str, Sequence[Union[Hashable, None]], None] = ...,
+        mode: _str = ...,
+        encoding: Union[_str, None] = ...,
+        compression: Union[_str, Mapping[str, str], None] = ...,
+        quoting: Union[int, None] = ...,
+        quotechar: _str = ...,
+        line_terminator: Union[_str, None] = ...,
+        chunksize: Union[int, None] = ...,
+        date_format: Union[_str, None] = ...,
```

Comment:
These `Union[..., None]` should probably be `Optional[...]`. It means the same, but we use Optional everywhere...

Reply:
Ok, tonight I'll made that change! I struggles because pandas doc use` Union[str, NoneType] `and I tried to stay as close to that as possible, but I agree, there should be consistency. Thanks you for the tips on black!